src/pretix/control/templates/pretixcontrol/event/plugins.html

```diff
@@ -11,74 +18,74 @@ <h1>{% trans "Installed plugins" %}</h1>
+                                                <span class="text-muted">{% trans "Not available" %}</span>
+                                            </div>
+                                        {% elif plugin.module in plugins_active %}
+                                            <div class="plugin-action flip">
+                                                <button class="btn btn-default{% if plugin.featured %} btn-lg{% endif %}" name="plugin:{{ plugin.module }}"
+                                                        value="disable">{% trans "Disable" %}</button>
                                             </div>
-                                        {% endif %}
-                                    </td>
-                                    <td class="text-right flip" width="20%">
-                                        {% if plugin.app.compatibility_errors %}
-                                            <button class="btn disabled btn-block btn-default"
-                                                    disabled="disabled">{% trans "Incompatible" %}</button>
-                                            {% elif plugin.restricted and not staff_session %}
-                                            <button class="btn disabled btn-block btn-default"
-                                                    disabled="disabled">{% trans "Not available" %}</button>
-                                            {% elif plugin.module in plugins_active %}
-                                            <button class="btn btn-default btn-block" name="plugin:{{ plugin.module }}"
-                                                    value="disable">{% trans "Disable" %}</button>
                                         {% else %}
-                                            <button class="btn btn-default btn-block" name="plugin:{{ plugin.module }}"
-                                                    value="enable">{% trans "Enable" %}</button>
+                                            <div class="plugin-action flip">
+                                                <button class="btn btn-primary{% if plugin.featured %} btn-lg{% endif %}" name="plugin:{{ plugin.module }}"
+                                                        value="enable">{% trans "Enable" %}</button>
+                                            </div>
                                         {% endif %}
-                                    </td>
-                                </tr>
-                            {% endfor %}
-                        </table>
+                            {% if plugin.featured %}
+                                    </div>
+                                </div>
+                            {% endif %}
+                            </div>
+                        {% endfor %}
                     </div>
                 </fieldset>
             {% endfor %}
         </div>
     </form>
-{% endblock %}
+{% endblock %}
```

Comment:
Why delete the EOF blank line?
It is a popular convention to keep one blank line at the end of file. Please configure your editor to respect it.